### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
Added : 

- Modal Top Component
- Modal Dialog Center Component
- Added Javascript To Open & Close Model

### CODE REVIEW

File: Components/Image/index.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.kash-img-fluid {
+  width: 100%;
+  /* height: 300px; */
```
Comment (Prabhav-Sharma) [inline Components/Image/index.css:3]: Remove comments which are not needed

File: Components/Modal/index.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+.kash-modal-header {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+}
+.kash-modal-header > .fa-times-circle {
+  font-size: large;
+  cursor: pointer;
+}
+.kash-modal-body {
+  padding: 1rem;
```
Comment (Prabhav-Sharma) [inline Components/Modal/index.css:21]: Use global variables for properties like margin, padding, and gap to maintain consistency across the website, something like --space-xsmall: 0.5rem

File: Components/Typography/index.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+              <ins
+                >This line of text is meant to be treated as an addition to the
+                document.</ins
+              >
+            </p>
+            <p><u>This line of text will render as underlined.</u></p>
+            <p>
+              <small
+                >This line of text is meant to be treated as fine print.</small
+              >
```
Comment (Prabhav-Sharma) [inline Components/Typography/index.html:110]: Use an extension like prettier or beautify to reformat indentation and code before pushing

File: Utilities/typography.css [hunk 4]
```diff
@@ -3,7 +3,7 @@
   margin: 0px 0px 0.35em;
   font-family: "Roboto", sans-serif;
   font-weight: 300;
-  font-size: 6rem;
+  font-size: 5rem;
   line-height: 1.167;
   letter-spacing: -0.01562em;
```
Comment (Prabhav-Sharma) [inline Utilities/typography.css:8]: How have you calculated such values for the line-height and letter spacing?

File: index.css [hunk 5]
```diff
@@ -19,7 +21,7 @@
 body {
   font-family: "Roboto";
   margin: 0;
-  height: 100vh;
+  /* height: 100vh; */
```
Comment (Prabhav-Sharma) [inline index.css:24]: Remove comment

**Review by Prabhav-Sharma — COMMENTED** [review]
Make sure you only change files pertaining to the component in a single PR.

File: Components/Image/index.css (hunk 1 above)
Comment (rehanmohiuddin) [reply Components/Image/index.css:3]: ya thaught of removing all comments at once

File: Utilities/typography.css (hunk 4 above)
Comment (rehanmohiuddin) [reply Utilities/typography.css:8]: did by trial and error & gone through various sites like bootstrap,material UI,chakra

File: Components/Typography/index.html (hunk 3 above)
Comment (rehanmohiuddin) [reply Components/Typography/index.html:110]: I have used prettier extension extension only auto saved like this

File: index.css (hunk 5 above)
Comment (rehanmohiuddin) [reply index.css:24]: noted

File: Components/Modal/index.css (hunk 2 above)
Comment (rehanmohiuddin) [reply Components/Modal/index.css:21]: ha i am using still forgetting sometimes slowly practicing

File: Components/Image/index.css (hunk 1 above)
Comment (rehanmohiuddin) [reply Components/Image/index.css:3]: my mistake while making this component observed responsive issue with image so did will remember next time

File: Components/Modal/index.html [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+      Modal({
+        modalID: "kash-modal",
+        modalBtnID: "kash-modal-open",
+        closeModalID: "kash-modal-close",
+      });
+      Modal({
+        modalID: "kash-modal-1",
+        modalBtnID: "kash-modal-open-1",
+        closeModalID: "kash-modal-close-1",
+      });
```
Comment (nerdyspook) [inline Components/Modal/index.html:154]: You can add this script in a js file, it will be easier to maintain and reuse.

**Review by nerdyspook — CHANGES_REQUESTED** [review]

File: Components/Modal/index.html (hunk 6 above)
Comment (rehanmohiuddin) [reply Components/Modal/index.html:154]: noted thanks

**Pull request merged**